Sources/Helpers/UIFont+DynamicTypeCustom.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIFont+DynamicTypeCustom.swift
+//  Pods
+//
+//  Created by Nikki Vergracht on 22/05/2017.
+//
+//
+
+import UIKit
+
+/// Available from iOS & tvOS 7.0 upwards.
+@available(iOS 7.0, tvOS 7.0, *)
+extension UIFont {
+    
+    /// Uses the `fontDescriptor` and `fontFamily` provided to return a custom UIFont compatible with the Dynamic Type selected in the user's settings.
+    private class func preferredFont(forFontDescriptor fontDescriptor: UIFontDescriptor, withFontFamily fontFamily: String) -> UIFont {
```

Comment:
I would make a more swift like function. 
`private class func preferredFont(for descriptor: UIFontDescriptor, with family: String) -> UIFont`